src/acom_music_box/music_box.py

```diff
@@ -279,6 +260,17 @@ def readConditionsFromJson(self, path_to_json):
             # Set initial conditions
             self.evolving_conditions = EvolvingConditions.from_config_JSON(
                 path_to_json, data, self.species_list, self.reaction_list)
+            
+        camp_path = os.path.join(
+        os.path.dirname(path_to_json),
+        box_model.config_file)
```

Comment:
```suggestion
        camp_path = os.path.join(
        os.path.dirname(path_to_json),
        self.config_file)
```

We don't need to pass the box model in because `self` **is** the box model